src/grt/src/fastroute/src/utility.cpp

```diff
@@ -1122,6 +1122,19 @@ void FastRouteCore::StNetOrder()
   std::stable_sort(
       tree_order_cong_.begin(), tree_order_cong_.end(), compareTEL);
 
+  // Set the 70% (or less) of non critical nets that doesn't have overflow
```

Comment:
Could you explain in more detail the idea of this update? What set of nets should have the highest priority?

Reply:
The  idea is to first ripup at least 30% of the nets(all non critical), with the highest priority to non critical nets with overflow, than the critical nets, and with the lowest priority the non critical nets without overflow. 
The non critical nets with overflow are ripped up first to decrease the chances of critical nets also getting ripped up, and if the do the should have more space to find routings that doesn't has a big deroute.